src/components/two-factor-authentication/HardwareTokenWorkflow.vue

```diff
@@ -50,8 +50,13 @@
     errorThrown.value = true;
     if (axios.isAxiosError(error)) {
       if (error.response && error.response.data.i18nKey) {
+        let i18nKey: string = error.response.data.i18nKey;
+        if (i18nKey === 'PI_UNAVAILABLE_ERROR') {
+          i18nKey = 'HARDWARE_TOKEN_SERVICE_FEHLER';
+        }
```

Comment:
Wieso nicht direkt ein Key `PI_UNAVAILABLE_ERROR` in de-DE.json schreiben?

Reply:
Die Fehlermeldung ist unterschiedliche je nachdem auf welcher Seite man sich befindet.